### PR TITLE
Question add form

### PR DESCRIPTION
**Pull request by chifgeri**

### CODE REVIEW

File: src/components/Main.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import { Switch, Route, Redirect } from 'react-router-dom';
-
+import AddQuestionContainer from '../containers/AddQuestionContainer';
 
 const Main = () => (
   <Switch>
     <Redirect exact from='/' to='/home' />
     <Route exact path='/home' />
+    <Route path='/q/add' component={AddQuestionContainer} />
```
Comment (danielgaldev) [inline src/components/Main.js]: /q/new

File: src/components/forms/AddQuestionForm.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+          label='Kérdés:'
+          value={title}
+          onChange={(e) => setTitle(e.target.value)}
+        />
+        <Form.TextArea
+          label='Leírás:'
+          value={details}
+          onChange={(e) => setDetails(e.target.value)}
+        />
+        <Form.Dropdown
```
Comment (danielgaldev) [inline src/components/forms/AddQuestionForm.js]: Tags are not a part of this milestone.

File: src/components/forms/AddQuestionForm.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+          selection
+          allowAdditions
+          options={tagOptions}
+          value={tags}
+          onChange={(e, data) => setTags(data.value)}
+          onAddItem={(event, data) => { /* TODO: Update tags remotely too */
+            setTagOptions([...tagOptions, {text: data.value, value: data.value}]);
+            setTags([...tags, data.value]);
+          }}
+        />
```
Comment (danielgaldev) [inline src/components/forms/AddQuestionForm.js]: There should be a checkbox for anonymous questions.

File: src/actions/questions.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+import { ADD_QUESTION } from './types';
+import history from '../history';
+
+export const addQuestion = ({title, details, tags}) => (
+
+  async (dispatch) => {
+    try {
+      const response = await axios.post('/api/v1/questions/', { title, text: details, tags });
+
+      if (response.status >= 200 && response.status < 300) {
```
Comment (ferencmarkizay) [inline src/actions/questions.js]: Why not just check for 201?

File: src/components/forms/AddQuestionForm.js (hunk 3 above)
Comment (danielgaldev) [reply src/components/forms/AddQuestionForm.js]: API fixed in https://github.com/DevTeamSCH/vikoverflow-backend/pull/31, `show_username` now available.

**Pull request merged**